documentation/general/general_setup.md

```diff
@@ -125,6 +125,7 @@ ones you might see used throughout the documentation:
 - [`httpie`](https://httpie.io/docs/cli) is a command line HTTP client. Like
   cURL but with a lot of conveniences and easier to remember. Try `http --help`
   for info.
+- ['psql'] (https://www.postgresql.org/docs/current/app-psql.html) is the PostgreSQL client
```

Comment:
Yea adding to @dhruvkb's comment, do you mind adding a line for `pgcli` and its documentation here as well?

Reply:
Sure. 

Is there some way to lint a single file? This doc is failing the lint test on the CI. I ran the lint command but it failed for a bunch of JS stuff.

Reply:
First, even if lint fails with a JavaScript error, it will make the necessary changes to the file. You can see them by running `git status` or `git diff`.

<img width="1080" alt="Screenshot 2024-07-30 at 4 03 50 PM" src="https://github.com/user-attachments/assets/1ff7d377-1511-4e01-814f-c4c5ebecced8">

Secondly, staging a particular file makes the lint job only run the hooks that are relevant for it, so the failing JS hooks will not run and you will be able to commit.

<img width="1080" alt="Screenshot 2024-07-30 at 4 06 34 PM" src="https://github.com/user-attachments/assets/77c0350e-ea1f-41f9-9774-d7e8660322fd">

Thirdly, you can run a single hook for a single file using `ov just lint <hook> <files>`, which in this case would be `ov just lint prettier`.